src/modules/collectionCuration/sagas.ts

```diff
@@ -0,0 +1,92 @@
+import { call, takeEvery } from '@redux-saga/core/effects'
+import { CONNECT_WALLET_SUCCESS } from 'decentraland-dapps/dist/modules/wallet/actions'
+import { BuilderAPI } from 'lib/api/builder'
+import { put } from 'redux-saga-test-plan/matchers'
+import {
+  approveCollectionCurationFailure,
+  ApproveCollectionCurationRequestAction,
+  approveCollectionCurationSuccess,
+  APPROVE_COLLECTION_CURATION_REQUEST,
+  fetchCollectionCurationFailure,
+  fetchCollectionCurationRequest,
+  FetchCollectionCurationRequestAction,
+  fetchCollectionCurationsFailure,
+  fetchCollectionCurationsRequest,
+  fetchCollectionCurationsSuccess,
+  fetchCollectionCurationSuccess,
+  FETCH_COLLECTION_CURATIONS_REQUEST,
+  FETCH_COLLECTION_CURATION_REQUEST,
+  pushCollectionCurationFailure,
+  PushCollectionCurationRequestAction,
+  pushCollectionCurationSuccess,
+  PUSH_COLLECTION_CURATION_REQUEST,
+  rejectCollectionCurationFailure,
+  RejectCollectionCurationRequestAction,
+  rejectCollectionCurationSuccess,
+  REJECT_COLLECTION_CURATION_REQUEST
+} from './actions'
+import { CollectionCuration, CurationStatus } from './types'
+
+export function* curationSaga(builder: BuilderAPI) {
```

Comment:
this saga should maybe be called `collectionCurationSaga` right?

Reply:
Yeah! updated to `collectionCurationSaga` :)